beacon-chain/p2p/discovery.go

```diff
@@ -358,7 +358,17 @@ func convertToSingleMultiAddr(node *enode.Node) (ma.Multiaddr, error) {
 	if err != nil {
 		return nil, errors.Wrap(err, "could not get peer id")
 	}
-	return multiAddressBuilderWithID(node.IP().String(), uint(node.TCP()), id)
+	return multiAddressBuilderWithID(node.IP().String(), "tcp", uint(node.TCP()), id)
```

Comment:
Might be better to declare constants for "tcp" and "udp" and use them here

Similar to `beacon-chain/p2p/topics.go`